accounting/localizations/mexico.rst

```diff
@@ -257,74 +372,41 @@ If the invoice is not paid.
 **Legal considerations**
 
 - A cancelled invoice will automatically cancelled on the SAT.
-- If you retry to use the same invoice after cancelled, you will have as much
-  cancelled CFDI as you tried, then all those xml are important to maintain a
-  good control of the cancellation reasons.
-- You must unlink all related payment done to an invoice on odoo before
-  cancel such document, this payments must be cancelled to following the same
-  approach but setting the "Allow Cancel Entries" in the payment itself.
-
-
-Payments (Just available for CFDI 3.3)
---------------------------------------
-
-To generate the payment complement you only need to follow the normal payment
-process in Odoo, this considerations to understand the behavior are important.
-
-#. To generate payment complement the payment term in the invoice must be
-   PPD, because It is the expected behavior legally required for
-   "Cash payment".
-
-   **1.1. How can I generate an invoice with payment term `PUE`?**
+- If you retry to use the same invoice after cancelled, you will have as much cancelled CFDI as you tried, then all
+  those xml are important to maintain a good control of the cancellation reasons.
+- You must unlink all related payment done to an invoice on odoo before cancel such document, this payments must be
+  cancelled to following the same approach but setting the "Allow Cancel Entries" in the payment itself.
 
-   `According to the SAT documentation`_ a payment is classified as ``PUE`` if
-   the invoice was agreed to be fully payed before the 17th of the next
-   calendar month (the next month of the CFDI date), any other condition
-   will generate a ``PPD`` invoice.
+Credit Notes
+------------
 
-   **1.2. How can I get this with Odoo?**
+# todo
```

Comment:
@nhomar we can remove this todo an  add it on new PR once the documentation for Credit Notes is ready.